addon_service/common/network.py

```diff
@@ -92,6 +92,8 @@ async def _try_send(self, request: HttpRequestInfo):
             request.http_method,
             _url,
             headers=await _private.get_headers(),
+            params=request.query,
+            json=request.json,
             # TODO: content
```

Comment:
Are these **TODO's** you currently have doable or if not do we need to include all of them if we won't be doing them?

Reply:
These TODO's had been here before me


I'd leave them as there might be need to include urlencoded data or something else.

Reply:
Oh okay, if they had been here before than leave it as it is